assets/eip-3267/contracts/BaseRestorableSalary.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: LGPL-3.0-or-later
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
For licensing change like this, ask the original code author for approval?

Reply:
Yes